src/components/DragDrop/DragDrop.tsx

```diff
@@ -78,60 +77,63 @@ export const DragDrop = ({
   /**
    * Target drag drop and drag drop inner DOM elements
    */
-  const dragDropInnerRef = useRef() as MutableRefObject<HTMLDivElement>;
-  const dragDropRef = useRef() as MutableRefObject<HTMLElement>;
+  const dragDropInnerRef = useRef<HTMLDivElement>(null);
+  const dragDropRef = useRef<HTMLElement | null>(null);
```

Comment:
Specifying the `| null` type makes the ref "mutable" from TS's perspective.

Not sure why the types are written like that, but that's how it is 🤷